Geometry_Engine/Compute/FitLine.cs

```diff
@@ -37,13 +37,13 @@ public static partial class Compute
 
         [Description("Fits a line into a set of points using Orthogonal Least Squares algorithm.")]
```

Comment:
```suggestion
        [Description("Fits a line into a set of points using Orthogonal Least Squares algorithm. Will return null if the number of non-duplicate points is less than two.")]
```

Maybe something like this if we don't want to raise an error?